csharp/src/Google.Protobuf/project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
```

Comment:
I think they need to be consumable with some reasonably new version without the need to update to some cutting edge version of mono (I think we should make it work on let's say mono 4+).

About buildability - that's not a strict requirement, but you need that to be able to run tests (see my other comments about travis and jenkins), unless you update the testing infrastructure to run tests is some other way.  Also, not being able to build and run unit tests on linux would seem like  a step backward.